entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/utils/SolrEntityUtils.java

```diff
@@ -0,0 +1,20 @@
+package eu.europeana.entitymanagement.solr.utils;
+
+import eu.europeana.entitymanagement.definitions.model.Agent;
+import eu.europeana.entitymanagement.definitions.model.Entity;
+import eu.europeana.entitymanagement.solr.model.SolrAgent;
+import eu.europeana.entitymanagement.solr.model.SolrEntity;
+
+public class SolrEntityUtils {
```

Comment:
this class doesn't seem to be needed, if not used please remove

Reply:
Removed.